site/app/templates/admin/admin_gradeable/AdminGradeableDates.twig

```diff
@@ -76,7 +76,7 @@
 
             <br />
 
-            <input data-testid="grade_inquiry_due_date" name="grade_inquiry_due_date" id="date_grade_inquiry_due" class="date_picker grade_inquiry_due_date" type="text" value="{{ gradeable.getGradeInquiryDueDate()|date(date_format, timezone_string) }}">
+            <input data-testid="date-grade-inquiry-due" name="grade_inquiry_due_date" id="date_grade_inquiry_due" class="date_picker grade_inquiry_due_date" type="text" value="{{ gradeable.getGradeInquiryDueDate()|date(date_format, timezone_string) }}">
```

Comment:
```suggestion
            <input data-testid="grade-inquiry-due-date" name="grade_inquiry_due_date" id="date_grade_inquiry_due" class="date_picker grade_inquiry_due_date" type="text" value="{{ gradeable.getGradeInquiryDueDate()|date(date_format, timezone_string) }}">
``` 
This is more of a consistency suggestion between the other date testids, but I can be persuaded to use different testids if they are all consistent.